paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/PaymentSelectionUpdater.kt

```diff
@@ -56,6 +56,9 @@ internal class DefaultPaymentSelectionUpdater @Inject constructor() : PaymentSel
             is PaymentSelection.Link -> {
                 state.linkState != null
             }
+            is PaymentSelection.ExternalPaymentMethod -> {
+                true
```

Comment:
I think this needs to look at `state.paymentMethodMetadata.externalPaymentMethods` to make sure the selected one exists in the current configuration.